### PR TITLE
Footer mac version init

### PR DESCRIPTION
**Pull request by Ntwalii**
Implemented the footer desktop version, working on responsiveness.

### CODE REVIEW

**Review by josueahadi — APPROVED** [review]
@Ntwalii @beylar the code looks good

**Review by Chartine02 — APPROVED** [review]

File: src/sections/footer.html [hunk 1]
```diff
@@ -1,4 +1,26 @@
-<div>
-    <h1>Footer</h1>
-    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam, quibusdam.</p>
+<div class="px-7 relative mt-36">
+    <img src="./src/images/Vector_7.svg" class="mx-auto sticky top-0 z-30" alt="">
```
Comment (Kingcedru) [inline src/sections/footer.html]: Add image description on the alt attribute.

File: src/sections/footer.html [hunk 2]
```diff
@@ -1,4 +1,26 @@
...
+
```
Comment (Kingcedru) [inline src/sections/footer.html]: Remove unnecessary spaces.

File: src/sections/footer.html [hunk 3]
```diff
@@ -1,4 +1,26 @@
...
+    <div class="flex gap-7 sm:flex-col lg:flex-row flex-col lg:justify-between md:gap-10 md:text-2xl font-medium md:mt-40 mb-4 mt-20">
+        <div class="flex flex-col lg:gap-28 justify-between md:flex-row gap-7 ">
+            <div class="flex flex-col ">
```
Comment (Kingcedru) [inline src/sections/footer.html]: `<div class="flex flex-col ">`
replace with:
`<div class="flex flex-col">`

The reason is to be constant in the spaces you put in your codes.

**Review by Kingcedru — CHANGES_REQUESTED** [review]
Try to be constant.

File: index.html [hunk 4]
```diff
@@ -11,7 +11,7 @@
     />
     <link rel="stylesheet" href="/dist/output.css" />
     <title>Website Project</title>
-  </head>
+  </head>~
```
Comment (Kingcedru) [inline index.html]: You can remove the ~ sign.

**Review by Kingcedru — CHANGES_REQUESTED** [review]
Correct also this mistake.

**Review by Kingcedru — APPROVED** [review]
Great

**Pull request merged**